test/metabase/api/setup_test.clj

```diff
@@ -306,8 +311,9 @@
           ;; In the non-test context, this is 'set' iff there is one or more users, and doesn't have to be toggled
           (reset! has-user-setup true)
           (is (setup/has-user-setup))
-          (mt/discard-setting-changes [site-name site-locale anon-tracking-enabled admin-email]
-            (is (= "The /api/setup route can only be used to create the first user, however a user currently exists."
+          ;; use do-with-setup* to delete the random user that was created
+          (do-with-setup* body
```

Comment:
wrapped this with `do-with-setup*` as a quick and easy way to clean up the user created by this test